src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -479,12 +485,32 @@ redeemADA :: WalletWebMode ssc m => SendActions m -> CWalletRedeem -> m CTx
 redeemADA sendActions CWalletRedeem {..} = do
     let base64rify = T.replace "-" "+" . T.replace "_" "/"
```

Comment:
@Martoon-00 are you sure its working exactly like `base64rify` ? Because it is essential we match it with how we did in https://github.com/input-output-hk/cardano-sl/pull/163/files#diff-fc6ae95807983ca40cedce171a560ba2R41 . See https://serokell.slack.com/archives/G1GD88UUX/p1490953691910513 . It was also done 4 days ago. I just don't want to use different encoding method. Some base64 url implementation ignore padded `=` at the end, some don't.